python/revng/pypeline/cli/project/daemon.py

```diff
@@ -0,0 +1,88 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import logging
+
+import click
+import uvicorn
+import yaml
+
+import revng
+from revng.pypeline.daemon.app import app
+from revng.pypeline.daemon.daemon import Daemon
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
```

Comment:
Add a `--production` option, should do the same as the `revng daemon` counterpart